plugin/src/main/cpp/include/extensions/openxr_fb_space_warp_extension_wrapper.h

```diff
@@ -0,0 +1,91 @@
+/**************************************************************************/
+/* Copyright (c) 2022-present Godot XR contributors (see CONTRIBUTORS.md) */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef OPENXR_FB_SPACE_WARP_EXTENSION_WRAPPER_H
+#define OPENXR_FB_SPACE_WARP_EXTENSION_WRAPPER_H
+
+#include <openxr/openxr.h>
+#include <godot_cpp/classes/open_xr_extension_wrapper_extension.hpp>
+#include <godot_cpp/classes/open_xrapi_extension.hpp>
+#include <map>
+
+using namespace godot;
+
+class OpenXRFbSpaceWarpExtensionWrapper : public OpenXRExtensionWrapperExtension {
+	GDCLASS(OpenXRFbSpaceWarpExtensionWrapper, OpenXRExtensionWrapperExtension);
+
+public:
+	static OpenXRFbSpaceWarpExtensionWrapper *get_singleton();
+
+	OpenXRFbSpaceWarpExtensionWrapper();
+	virtual ~OpenXRFbSpaceWarpExtensionWrapper() override;
+
+	godot::Dictionary _get_requested_extensions() override;
+
+	uint64_t _set_system_properties_and_get_next_pointer(void *p_next_pointer) override;
+	uint64_t _set_projection_views_and_get_next_pointer(int p_view_index, void *p_next_pointer) override;
+
+	void _on_session_created(uint64_t p_instance) override;
+	void _on_session_destroyed() override;
+	void _on_state_ready() override;
+	void _on_main_swapchains_created() override;
+	void _on_pre_render() override;
+	void _on_post_draw_viewport(const RID &p_render_target) override;
+
+	bool is_available();
+
+	void enable_space_warp(bool p_enable);
```

Comment:
I wonder if `set_space_warp_enabled()` or `set_enable_space_warp()` would be more Godot-ish? We do have a handful of `enable_*()` functions, but I think the other two variations are more common.

Reply:
I agree, I've updated it to be `set_space_warp_enabled`!